### ISSUE TITLE
Wrong link in README

### ISSUE THREAD

**Issue by ameliendeshams** [vrchat-community/ClientSim#25]
Hi,

There is a mistake on the link ;

```
  - Press the + button one last time and add this url: `https://github.com/vrchat-community/ClientSim.git?path=/ClientSim_UnityProject/Packages/com.vrchat.ClientSim#vcc`
```
that returns a 404 error, the correct link is ;

```
https://github.com/vrchat-community/ClientSim?path=/Packages/com.vrchat.ClientSim#vcc
```

**Comment by Toxic-Cookie** [comment]
Your updated link still seems to be incorrect. The one that worked for me is this:
```
https://github.com/vrchat-community/ClientSim.git#1.1.0
```

**Comment by momo-the-monster** [comment]
Fixed in #21
